codespell_lib/data/dictionary_en-GB_to_en-US.txt

```diff
@@ -121,6 +121,7 @@ legalised->legalized
 legalises->legalizes
 legalising->legalizing
 licence->license
+licenced->licensed
```

Comment:
I think this should be licensed->licensed i.e. a No-Op

Reply:
you just wrote `licensed` twice :point_up:

Reply:
Yes, I think for a quick check that the British English is `licensed` too, unless you can point me to a source I've missed. I did try and double-check online...

Reply:
Sorry, I probably didn't explain very well, what I meant is that the British word is also licensed, so as you say @vikivivi it should be removed because it's a no-op. I think our tests should catch that anyway.